### PR TITLE
Trial _Project

### PR DESCRIPTION
**Pull request by shahnawaza75**
Build with html css 
Added one image also

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+  **Authors**
+  :silhouette: **Shahnawaz Ali**
+- GitHub: [@githubhandle](https://github.com/shahnawaza75)
+- Twitter: [@twitterhandle](https://twitter.com/RjShahnawaza75)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shahnawaz-ali-a24b72204)
+  **:handshake: Contributing**
+  Contributions, issues, and feature requests are welcome!
+  Give a :star:️ if you like this project!
+  **:memo: License**
+  This project is (./MIT.md) licensed.
```
Comment (shahnawaza75) [inline README.md:20]: **changes required***

It would be nice if you update your new MIT license as your current MIT license is expired.

**Review by shahnawaza75 — COMMENTED** [review]
**changes required ♻️ 

Please check my comments below and make all requuired changes.

Submit for a re-review when you are done.

Thanks 😁

File: README.md (hunk 1 above)
Comment (roshan-bajgain) [reply README.md:20]: - [ ] It would be nice you state a reason(s) for the change you are asking the student to make in a polite and friendly way.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @shahnawaza75 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] According to the project requirements the student should write at least 2 comments in the review.  So kindly update at least two comments.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+**Built With**
+
+- HTML
+- CSS
+- VS Code
+  **Authors**
+  :silhouette: **Shahnawaz Ali**
+- GitHub: [@githubhandle](https://github.com/shahnawaza75)
+- Twitter: [@twitterhandle](https://twitter.com/RjShahnawaza75)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shahnawaz-ali-a24b72204)
```
Comment (shahnawaza75) [inline README.md:15]: It will be nice if you add your Username to your link text for your socials. `@shahnawaza75`

Comment (topeogunleye) [reply README.md:15]: - [ ] Nice job being polite, giving reasons for the requested change, and phrasing your comments as suggestions

File: README.md (hunk 1 above)
Comment (topeogunleye) [reply README.md:20]: - [ ] Nice job being polite, giving reasons for the requested change, and phrasing your comments as suggestions

**Review by topeogunleye — APPROVED** [review]
### Hi @braincee,
Great job on this project review! 👍🏻
# Status: Approved ✔️
Excellent job on implementing all the changes requested. Keep it up 👏
Your project review is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

### To Highlight
- [x]  Added descriptive inline comments.
- [x]  Included an explanation, a "why", for each requested change.
- [x]  Great use of emojis to make your code review summary more human-friendly 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Congratulations-Memes-FEATURED

**Pull request merged**